packages/viz/__tests__/Layer.spec.ts

```diff
@@ -46,7 +53,7 @@ describe('Layer', () => {
         'https://notdefaultserver.com'
       );
 
-      const layer = new Layer(DEFAULT_DATASET, { credentials });
+      const layer = new Layer(DEFAULT_DATASET, {}, { credentials });
```

Comment:
I guess this 2nd param is Style. Should we consider adding it to the 'options' bag?

Reply:
Yes, it is our style definition. But I think we should not add it to our options object, just for simplicity, it's much better to specify it like:
```js
const layer = new carto.viz.Layer(DEFAULT_DATASET, carto.viz.colorBinsStyle('column'))
```
rather than:
```js
const layer = new carto.viz.Layer(DEFAULT_DATASET, {
  style: carto.viz.colorBinsStyle('column')
})
```

What we can do is to detect which parameter comes second and then change our behaviour. I'm gonna write it down in Clubhouse.

Reply:
Mmm, I see. Well I won't bother right now with 'auto-detecting'. Let's leave it as it is.